src/application-base/private/_contains-polyfill.js

```diff
@@ -0,0 +1,19 @@
+// Source: https://github.com/Financial-Times/polyfill-service/pull/183/files
```

Comment:
can we have these same files moved to single common folder for entire terra-application package

Reply:
Done 8e05b49